.changeset/small-drinks-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
+---
+
+Adding a few more background color atomics. Changing card's title background color to transparent.
```

Comment:
List which color atomics are added? Also, we still have no color documentation :(.

Reply:
I hope I can fix that next semester. 🤞🏻